spec/features/controllers/api/v1/animals_controller_spec.rb

```diff
@@ -14,13 +14,25 @@
     photo_path: "./spec/support/images/photo.png"
   ) }
 
+  let!(:user1) {User.create(
+    username: "josh",
+    password: "111111",
+    email: "jgorin@conncoll.edu"
+  )}
```

Comment:
A user is created, but not used? You would need this in the spec if a user needed to be logged in to see the page/create a review etc.

Reply:
We needed to create a user object in order to create a new review

```
  let!(:review) {Review.create(
    title: "bad",
    description: "soo bad",
    rating: 1,
    animal: first_animal,
    user: user1
  )}
```